osf/models/institution.py

```diff
@@ -18,18 +18,37 @@ class Institution(Loggable, base.ObjectIDMixin, base.BaseModel):
     # /TODO DELETE ME POST MIGRATION
 
     # TODO Remove null=True for things that shouldn't be nullable
-    banner_name = models.CharField(max_length=255, null=True, blank=True)
+
+    name = models.CharField(max_length=255)
+    description = models.TextField(blank=True, null=True, default='')
+
+    # TODO Could `banner_name` and `logo_name` be a FilePathField?
+    # Both banner_name and logo_name are the name of the file under the `banners/` or `shields/` folder in
+    # `website/static/img/institutions/`. They cannot be null or empty, otherwise OSF dashboard page and institution
+    # project page won't finish loading.
+    banner_name = models.CharField(max_length=255, blank=True, default='')
+    logo_name = models.CharField(max_length=255, blank=True, default='')
+
+    # the protocol used to delegate authentication: `CAS`, `SAML`, `OAuth`, e.t.c
+    # only institutions with a delegation protocol (not null and not empty) shows up on institution login page
+    delegation_protocol = models.CharField(max_length=255, null=True, blank=True)
+
+    # login_url and logout_url can be null or empty
     login_url = models.URLField(null=True, blank=True)
-    contributors = models.ManyToManyField(settings.AUTH_USER_MODEL,
-                                          through=InstitutionalContributor,
-                                          related_name='institutions')
+    logout_url = models.URLField(null=True, blank=True)
+
     domains = fields.ArrayField(models.CharField(max_length=255), db_index=True, null=True, blank=True)
     email_domains = fields.ArrayField(models.CharField(max_length=255), db_index=True, null=True, blank=True)
-    logo_name = models.CharField(max_length=255, null=True)  # TODO: Could this be a FilePathField?
-    logout_url = models.URLField(null=True, blank=True)
-    name = models.CharField(max_length=255)
 
-    description = models.TextField(blank=True, default='', null=True)
+    # TODO what is this field?
```

Comment:
Oh, I thought you were talking about description.

It holds the list of contributors to the institution. People with permissions to contribute to the institution, assign projects to it, etc. It also isn't new. There is a contributors on Node that is what was used in modm.